components/developer/ccache/Makefile

```diff
@@ -52,3 +52,4 @@ REQUIRED_PACKAGES += library/zlib
 REQUIRED_PACKAGES += SUNWcs
 REQUIRED_PACKAGES += system/library
 REQUIRED_PACKAGES += system/library/math
+REQUIRED_PACKAGES += file/gnu-coreutils
```

Comment:
The new required packages makes an updated pkg5 file necessary.

Reply:
Strange, thought I added that, sorry